app/controllers/api/teams.php

```diff
@@ -341,7 +341,17 @@
                     'emailVerification' => false,
                     'status' => Auth::USER_STATUS_UNACTIVATED,
                     'password' => Auth::passwordHash(Auth::passwordGenerator()),
-                    'passwordUpdate' => \time(),
+                    /** 
+                     * Set the password update time to 0 for users created 
+                     * with a team invite. This will allow us to distinguish b/w 
+                     * new and old appwrite users. 
+                     * if (passwordUpdate == 0) {
+                     *      // new appwrite user created with oauth/anonymous/team invite
+                     * } else {
+                     *      // existing appwrite user
+                     * } 
+                     */
+                    'passwordUpdate' => 0,
```

Comment:
I think we can make this comment a bit smaller and less verbose. A one liner might be enough :-)

Reply:
I have updated this.. I hope it's better. But we should be strict about this. Let me know if this needs to be changed